texttospeech/cloud-client/resources/hello.ssml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
```

Comment:
Simplify this :)

For out samples, let's just use simple SSML:

```xml
<speak>Hello there.</speak>
```

Per request from product :)